3Homework03.03.22/CompressorLZW.cs

```diff
@@ -0,0 +1,137 @@
+using System;
+using System.Collections;
+
+namespace LZW.Solution;
+
+public class CompressorLZW
+{
+    public void ZipFile(string path)
+    {
+        byte[] file = File.ReadAllBytes(path);
+        byte[] buffer = new byte[4];
```

Comment:
```suggestion
        var buffer = new byte[4];
```
Не надо писать один тип дважды